azurerm/resource_arm_service_fabric_cluster.go

```diff
@@ -78,6 +78,31 @@ func resourceArmServiceFabricCluster() *schema.Resource {
 				Set:      schema.HashString,
 			},
 
+			"azure_active_directory": {
+				Type:     schema.TypeList,
+				Optional: true,
+				MaxItems: 1,
```

Comment:
if this block is added/removed we need to trigger a recreation of this resource - as such can we also add `ForceNew: true` here?

Reply:
Thanks, added the line.